Dockerfile_integration

```diff
@@ -15,7 +15,9 @@ COPY settings.gradle.kts .
 COPY gradle.properties .
 RUN $GRADLE build && $GRADLE test
 
-RUN apt-get update && apt-get install -y gosu && rm -rf /var/lib/apt/lists/*
+RUN apk update \
+    && apk add su-exec \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
That last line can go now that we're not using `apt`

Reply:
Good catch, will change